apps/wallet/src/ui/app/pages/home/tokens/SupplyIncreaseVestingStakingDialog.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Button, Dialog, DialogContent, DialogBody, Header } from '@iota/apps-ui-kit';
+import { Theme, useTheme } from '@iota/core';
+import SupplyIncreaseVestingStakingImage from '_assets/images/vested_staking_dialog.png';
+import SupplyIncreaseVestingStakingDarkImage from '_assets/images/vested_staking_dialog_darkmode.png';
+
+interface SupplyIncreaseVestingStakingDialogProps {
+    open: boolean;
+    setOpen: (isOpen: boolean) => void;
+}
+
+export function SupplyIncreaseVestingStakingDialog({
+    open,
+    setOpen,
+}: SupplyIncreaseVestingStakingDialogProps) {
+    const { theme } = useTheme();
+
+    const imgSrc =
+        theme === Theme.Dark
+            ? SupplyIncreaseVestingStakingDarkImage
+            : SupplyIncreaseVestingStakingImage;
+
+    function navigateToDashboard() {
+        window.open('https://wallet-dashboard-iota1.vercel.app', '_blank');
```

Comment:
we could use a shared constant to save the dashboard URL. We will also need it for the migration dialog